auth_session_timeout/data/ir_config_parameter_data.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+    (c) 2015 ACSONE SA/NV, Dhinesh D
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo noupdate="1">
+    <record id="inactive_session_time_out_delay" model="ir.config_parameter">
+        <field name="key">inactive_session_time_out_delay</field>
+        <field name="value">7200</field>
+    </record>
+    <record id="inactive_session_time_out_ignored_url" model="ir.config_parameter">
+        <field name="key">inactive_session_time_out_ignored_url</field>
+        <field name="value">/calendar/notify,/websocket</field>
```

Comment:
IMO, a migration script needs to be added to change the config parameter from longpolling/poll to websocket

Reply:
Thanks for your contribution @bizzappdev !